test/commands/InRadius.txt

```diff
@@ -1,315 +1,460 @@
   [ count patches in-radius 14 ] of patch 0 min-pycor => 121
-
-InRadiusLargeRadius2_3D
-  O> resize-world 0 10 0 10 0 10
-  O> __change-topology true true
-  [ count patches in-radius 14 ] of patch 0 0 0 => 1331
-  [ count patches in-radius 14 ] of patch 3 0 0 => 1331
-  [ count patches in-radius 14 ] of patch max-pxcor 0 0 => 1331
-  [ count patches in-radius 14 ] of patch 0 min-pycor 0 => 1331
+  [ count turtles in-radius 1 ] of patch 0 0 => 10
+  [ count turtles in-radius 1 ] of patch max-pxcor max-pycor => 0
 
 InRadiusLargeRadius3_2D
   O> resize-world 0 11 0 11
+  O> create-turtles 10
   O> __change-topology true true
   [ count patches in-radius 14 ] of patch 0 0 => 144
   [ count patches in-radius 14 ] of patch 3 0 => 144
   [ count patches in-radius 14 ] of patch max-pxcor 0 => 144
   [ count patches in-radius 14 ] of patch 0 min-pycor => 144
+  [ count turtles in-radius 1 ] of patch 0 0 => 10
+  [ count turtles in-radius 1 ] of patch max-pxcor max-pycor => 0
   O> __change-topology false false
   [ count patches in-radius 16 ] of patch 0 0 => 144
   [ count patches in-radius 16 ] of patch 3 0 => 144
   [ count patches in-radius 16 ] of patch max-pxcor 0 => 144
   [ count patches in-radius 16 ] of patch 0 min-pycor => 144
+  [ count turtles in-radius 1 ] of patch 0 0 => 10
+  [ count turtles in-radius 1 ] of patch max-pxcor max-pycor => 0
   O> __change-topology true false
   [ count patches in-radius 16 ] of patch 0 0 => 144
   [ count patches in-radius 16 ] of patch 3 0 => 144
   [ count patches in-radius 16 ] of patch max-pxcor 0 => 144
   [ count patches in-radius 16 ] of patch 0 min-pycor => 144
+  [ count turtles in-radius 1 ] of patch 0 0 => 10
+  [ count turtles in-radius 1 ] of patch max-pxcor max-pycor => 0
   O> __change-topology false true
   [ count patches in-radius 16 ] of patch 0 0 => 144
   [ count patches in-radius 16 ] of patch 3 0 => 144
   [ count patches in-radius 16 ] of patch max-pxcor 0 => 144
   [ count patches in-radius 16 ] of patch 0 min-pycor => 144
-
-InRadiusLargeRadius3_3D
-  O> resize-world 0 11 0 11 0 11
-  O> __change-topology true true
-  [ count patches in-radius 14 ] of patch 0 0 0 => 1728
-  [ count patches in-radius 14 ] of patch 3 0 0 => 1728
-  [ count patches in-radius 14 ] of patch max-pxcor 0 0 => 1728
-  [ count patches in-radius 14 ] of patch 0 min-pycor 0 => 1728
+  [ count turtles in-radius 1 ] of patch 0 0 => 10
+  [ count turtles in-radius 1 ] of patch max-pxcor max-pycor => 0
 
 InRadiusLargeRadius4_2D
   O> resize-world -10 0 -10 0
+  O> create-turtles 10
   O> __change-topology true true
   [ count patches in-radius 14 ] of patch 0 0 => 121
   [ count patches in-radius 14 ] of patch -3 0 => 121
   [ count patches in-radius 14 ] of patch max-pxcor 0 => 121
   [ count patches in-radius 14 ] of patch 0 min-pycor => 121
+  [ count turtles in-radius 1 ] of patch -10 -10 => 0
+  [ count turtles in-radius 1 ] of patch max-pxcor max-pycor => 10
```

Comment:
Our code special-cases radius 2, but we have no test cases which exercise that. Let's add some.